common/src/main/java/com/microsoft/identity/common/internal/migration/TokenCacheItemMigrationAdapter.java

```diff
@@ -136,7 +136,7 @@
      * @throws ClientException
      * @throws IOException
      */
-    public static boolean tryFociTokenWithGivenClientId(@NonNull final BrokerOAuth2TokenCache brokerOAuth2TokenCache,
+    public static boolean tryFociTokenWithGivenClientId(@SuppressWarnings("rawtypes") @NonNull final BrokerOAuth2TokenCache brokerOAuth2TokenCache,
                                                         @NonNull final String clientId,
```

Comment:
Use  @SuppressWarnings(rawtype_warning) as you did everywhere.

Reply:
Sure, I'll do.

Reply:
I have taken care of this. Thanks